src/test/java/nl/tudelft/pixelperfect/event/EventParametersTest.java

```diff
@@ -111,6 +122,14 @@ public void testEqualsDifferentSizes() {
   }
 
   /**
+   * An EventParameters instance should no be considered equal to something of another type.
```

Comment:
You're right, but null and a different type are handled exactly the same way: the instanceof yields false. It is just for the line coverage, so it really doesn't matter whether it is null or an instance of another class.